switch_name: "roller_limit_switch"
too_close_zone: 1.5